src/components/header/index.js

```diff
@@ -35,12 +37,16 @@ const Header = () => {
                     <Card>
                         <section className="post-details">
                             <div className="profile-icon">
-                                <p>AJ</p>
+                                <p>{user.firstName[0].toUpperCase()}{user.lastName[0].toUpperCase()}</p>
                             </div>
 
                             <div className="post-user-name">
-                                <p>Alex Jameson</p>
-                                <small>Software Developer, Cohort 3</small>
+                                <p>{user.firstName} {user.lastName}</p>
+                                {user.role === 'TEACHER' ?
+                                    <small>Teacher</small>
+                                :
```

Comment:
I feel like the semi-colon should be on line 46 after `</small>`

Reply:
By my understanding, this is part of a ternary function which is just spread across a few lines. So it's ok as is?